src/Core/SuperMemoAssistant.Core/SuperMemo/Common/Elements/ElementRegistryBase.cs

```diff
@@ -246,12 +246,12 @@ public IEnumerator<IElement> GetEnumerator()
 
     public IEnumerable<IElement> FindByName(Regex regex)
     {
-      return Elements.Values.Where(e => regex.IsMatch(e.Title)).Select(e => (IElement)e).ToList();
+      return Elements.Values.Where(e => regex.IsMatch(e.Comment)).Select(e => (IElement)e).ToList();
```

Comment:
I assume that comment registry is not title, cause my YT collection for example has no comment registry but has titles.